CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 
 ### General
 
+* Made lint check for parameters defaults stricter [[#992](https://github.com/nf-core/tools/issues/992)]
+    * Defaults must now match the variable type specified in the schema
+    * If you want the parameter to not have a default value, use `null`
+    * Strings set to `false` or an empty string in `nextflow.config` will now fail linting
```

Comment:
😍 Would it be also nice that if there is a default value defined in the `nextflow_schema.json` file the lint gives a warning (maybe an error is too much) when the value does not match the value in the `nextflow.config`?

Reply:
I thought it already gave an error? If not then yeah it should do. I think error is better than warning in this case as it's pretty bad (and can ignore it in edge cases).

Reply:
Ok, implemented in 58c37cf